portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,28 +27,23 @@
 public class DataServlet extends HttpServlet {
 
   ArrayList<String> comments = new ArrayList<String>();
```

Comment:
private access modifier?

Reply:
Do I need to change the public to private?

Reply:
private